README.md

```diff
@@ -7,8 +7,5 @@ Welcome to [OpenLayers 3](http://openlayers.org/)!
 ## Examples
 Check out the [hosted examples](http://openlayers.org/en/master/examples/), the [workshop](http://openlayers.org/ol3-workshop/) or poke around the evolving [API docs](http://openlayers.org/en/master/apidoc/).
 
-## Questions
```

Comment:
Personally I'd keep this section in the README since it's more likely to get viewed.
I think the chance of people clicking a link in the README is pretty low.